packages/front-end/pages/metrics.tsx

```diff
@@ -60,10 +60,14 @@ const MetricsPage = (): React.ReactElement => {
     }),
     [getDatasourceById]
   );
+  const filteredMetrics = metrics.filter((m) => {
+    if (!project) return true;
+    return m?.projects?.includes(project);
```

Comment:
Is this right?  If a metric is not tagged to any projects, it should always be included, right?